### ISSUE TITLE
Linter doesn't detect errors when describing config in package.json

### ISSUE THREAD

**Issue by Haroenv** [steelbrain/linter#1254]
As said in earlier issues, this linter relies on the presence of `.eslint.json` in project root. I've got: 

```json
  "eslintConfig": {
    "env": {
      "es6": true
    },
    "extends": "xo-space/browser"
  },
```

in `package.json`, but it doesn't seem to be detected.

**Comment by Arcanemagus** [comment]
Did you mean to file this on [linter-eslint](https://github.com/AtomLinter/linter-eslint)? Marking this as closed since Linter has nothing to do with how the providers work (with regards to this at least).
